conjure-java-server-verifier/src/test/java/com/palantir/conjure/java/verification/server/undertest/EchoUndertowAutoDeserializeService.java

```diff
@@ -0,0 +1,500 @@
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.OptionalUuidAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.RawOptionalExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.ReferenceAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.RidAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.RidExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SafeLongAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SafeLongExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetAnyAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetBearerTokenAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetBinaryAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetBooleanAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetDateTimeAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetDoubleAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetDoubleExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetIntegerAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetOptionalAnyAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetRidAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetSafeLongAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetStringAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetStringExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SetUuidAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.SnakeCaseObjectExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.StringAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.StringExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.UuidAliasExample;
+import com.palantir.conjure.java.com.palantir.conjure.verification.types.UuidExample;
+
+/**
+ * Simple implementation of {@link UndertowAutoDeserializeService}, mirroring the behavior of a proxy using
+ * {@link EchoResourceInvocationHandler}. This implementation is used instead of a proxy, due to difficulties calling
+ * default methods on the non-private-accessible interface {@link UndertowAutoDeserializeService#endpoints}.
+ */
+final class EchoUndertowAutoDeserializeService implements UndertowAutoDeserializeService {
```

Comment:
Builds will fail when conjure-verifier is upgraded since the methods on AutoDeserializeService may change. I have an internal link with an example of default method invocations that works on all supported jres, see github.p.b/g/conjure-java-service-adapter/pull/242

Reply:
That's a good solution. Updated the InvocationHandler proxy to handle default methods and removed the `EchoUndertowAutoDeserializeService`.